### PR TITLE
feat: fill defaults from function signatures into .cfg (issue #426)

### PR DESCRIPTION
**Pull request by MartinBernstorff**
Unblocked by Explosion in https://github.com/Aarhus-Psychiatry-Research/psycop-common/issues/426

### CODE REVIEW

**Comment by MartinBernstorff** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #638** <a href="https://app.graphite.dev/github/pr/Aarhus-Psychiatry-Research/psycop-common/638?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Aarhus-Psychiatry-Research/psycop-common/638?utm_source=stack-comment).

File: psycop/common/model_training_v2/config/config_utils.py [hunk 1]
```diff
@@ -14,15 +14,21 @@
 populate_baseline_registry()
 
 
-def load_baseline_config(config_path: Path) -> BaselineSchema:
-    """Loads the baseline config from disk and resolves it."""
+def resolve_and_fill_config(config_path: Path) -> dict[str, Any]:
```
Comment (KennethEnevoldsen) [inline psycop/common/model_training_v2/config/config_utils.py:17]: add flag overwrite_config.

Comment (KennethEnevoldsen) [reply psycop/common/model_training_v2/config/config_utils.py:17]: Could imagine you might want to disable the behaviour (it is nice to know that the config is being overwritten).

**Comment by MartinBernstorff** [comment]
Good point! Since it doesn't override values which already exist, perhaps something like `fill_cfg_with_defaults_from_callable`?

**Comment by KennethEnevoldsen** [comment]
`fill_cfg_with_defaults`

**Pull request merged**